packages/mdc-theme/README.md

```diff
@@ -11,6 +11,21 @@ path: /catalog/theme/
 
 The Material Design color system can be used to create a color scheme that reflects your brand or style.
 
+The colors in this module are derived from three theme colors:
```

Comment:
I think the OP is referring to missing CSS variable names in our docs that were removed in this commit (https://github.com/material-components/material-components-web/commit/9394b9dd1e735b29e390a92a00d9ce08b2f50cfb)

For example, documentation for `--mdc-theme--text-primary-on-light` CSS custom property. We should at least list all possible replacement for `TEXT_STYLE` in the table every time when we mention `TEXT_STYLE` (ie., primary, secondary, hint, disabled and icon).

Reply:
I believe the different options are in this PR between lines 22 and 26. Is there one that I'm missing?

Reply:
I was referring to these diffs specifically: https://github.com/material-components/material-components-web/commit/9394b9dd1e735b29e390a92a00d9ce08b2f50cfb#diff-7f578dcc2f8f072f080e3144e80a3e60L96

> A note about <TEXT_STYLE>, <TEXT_STYLE> represents the lowercase name of the text styles listed above, e.g. hint.

We've a note about `TEXT_STYLE` in our docs but it is not very clear which section we're referring to when we say **_listed above_**. It would be easier if we specify possible options for `TEXT_STYLE` next it it.

Reply:
Yes you're right...I updated it. WDYT?